### DISCUSSION TITLE
[Question]: Is it feasible to switch OpenAI Assistants within the same conversation?

### DISCUSSION THREAD

**Discussion by yinheli** [danny-avila/LibreChat#3305] [Q&A] [answered]
### What is your question?

Is it feasible to switch OpenAI's assistants within the same conversation? I see that the system currently allows such an operation. I wonder if this operation will cause confusion in the context. Are there any best practices in this regard? Or should we prompt to start a new conversation when switching assistants? 

### More Details

No

### What is the main subject of your question?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
It is fine to switch the assistant, this is allowed by design. Your context is primarily determined by the current context window and chat history.
